.eslintrc

```diff
@@ -0,0 +1,68 @@
+{
+  "env": {
+    "node": true,
+    "builtin": true
+  },
+  "rules": {
+    "block-scoped-var": 2,
```

Comment:
Looks generally good - so far lacking support for checking indentation and trailing whitespace